ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+Vampire_Names = "dra drac ula cula fang nosf eratu"
+Allergies = %w(peanuts sunshine wheat beestings)
+Employee_Roster = []
+
+
+def vamp_namer(name)
```

Comment:
Instead of splitting a parameter as the input for vampire name and comparing to split string - you can check if two strings are equal by using:
```
name = gets.chomp
name.downcase == "drake cula" || name.downcase == "tu fang" 
```